bootstrap/src/main/kotlin/jp/co/soramitsu/notary/bootstrap/config/BootstrapConfig.kt

```diff
@@ -0,0 +1,18 @@
+package jp.co.soramitsu.notary.bootstrap.config
```

Comment:
Can we change package naming? We are moving to `com.d3`

Reply:
This service will be separated from notary soon. It is designed to be common soramitsu bootstrap service. So package naming should not be project specific. So will change to 
jp.co.soramitsu.bootstrap